src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -967,6 +981,20 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
   glb_policy->args = grpc_channel_args_copy_and_add_and_remove(
       args->args, args_to_remove, GPR_ARRAY_SIZE(args_to_remove), &new_arg, 1);
 
+  /* Fallback is off */
+  glb_policy->fallback = false;
+
+  /* Copy the backend server addresses from the resolver for fallback */
+  grpc_lb_addresses *backends = grpc_lb_addresses_create(
+      addresses->num_addresses - num_grpclb_addrs, NULL);
+  for (size_t addr_copied = 0, i = 0; i < addresses->num_addresses; ++i) {
+    if (!addresses->addresses[i].is_balancer) {
+      memcpy(backends->addresses + addr_copied++, addresses->addresses + i, 
```

Comment:
I just realized that maybe it's better to add a `grpc_lb_address_set` function instead of `grpc_lb_address_copy` so that 

- we don't need to modify `grpc_lb_addresses_create` which already allocates the memory for the array of `grpc_lb_address`es,
- and the final array of `grpc_lb_address`es can be kept a continuous array.